extensions/amp-date-picker/0.1/test/integration/test-integration-dates-attributes.js

```diff
@@ -18,8 +18,7 @@ import * as lolex from 'lolex';
 import {htmlFor} from '../../../../../src/static-template';
 import {poll} from '../../../../../testing/iframe';
 
-const config = describe.configure()
-    .ifChrome().skipChromeDev().skipSinglePass();
+const config = describe.configure().skipSinglePass();
```

Comment:
Keep `ifChrome()` - it should allow the test to run on both stable and beta versions.

Reply:
Done